### PR TITLE
introduce 'known issues' to documentation

### PR DESCRIPTION
**Pull request by BIGjuevos**
We should let people know in the documentation that we are aware of certain problems, and if available what the fix is.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
:+1:  what do you think  @istr ? I fear a possible duplicate with faq section ?

**Comment by BIGjuevos** [comment]
Is it worth putting this as a sub section under the FAQ docs?  Or just converting this into FAQ style questions?

Though, putting it under FAQ feels like shoving a square peg into a round hole to me.

**Comment by mickaelandrieu** [comment]
Finaly, thinking about it make me feel it's a bad idea to put it in FAQ section.
Can you add a link to faq in the installation section ?

**Comment by istr** [comment]
> I fear a possible duplicate with faq section ?

IMHO duplication would hurt less than omission. I am not sure, though, if it is worth duplicating what is already tracked in github tickets. Maybe we stick with the policy to document only "known issues" there that we deem largely unresolvable (i.e. we cannot solve by means of a bug-fix or work-around), like the one @BIGjuevos started with.

tl;dr -- :+1: to merge this.

**Comment by mickaelandrieu** [comment]
thanks @BIGjuevos
